futurenhs.app/components/_pageLayouts/GroupLayout/index.tsx

```diff
@@ -23,15 +24,14 @@ export const GroupLayout: (props: Props) => JSX.Element = ({
     pageTitle,
     ...rest
 }) => {
-    /**
-     * TODO: Determine whether user has access to group when back-end permissions work is complete
-     */
-    const hasAccessToGroup: boolean = true
+
+    const router: any = useRouter()
+    const isAboutPage: boolean = router.asPath === routes.groupAboutRoot
```

Comment:
Recommend isAboutRoute rather than page, feels like too many terms for one thing going on in this line

Reply:
Done